sdk/cosmos/azure-cosmos/azure/cosmos/_models.py

```diff
@@ -0,0 +1,54 @@
+# The MIT License (MIT)
+# Copyright (c) 2023 Microsoft Corporation
+
+from typing import Any
+try:
+    from typing import Literal
+except ImportError:
+    from typing_extensions import Literal
+
+
+class BatchOperationResponse:
+    operation: dict
+    """Operation that was sent as part of the batch request."""
+    operation_response: dict
+    """The operation response."""
+    kind: Literal["BatchOperationResponse"] = "BatchOperationResponse"
+    """Error kind - "BatchOperationResponse"."""
+
+    def __init__(self, **kwargs: Any) -> None:
+        self.operation = kwargs.get("operation", None)
+        self.operation_response = kwargs.get("operation_response", None)
+        self.kind: Literal["BatchOperationResponse"] = "BatchOperationResponse"
+
+    def __repr__(self) -> str:
+        return (
+            f"OperationResult(operation={self.operation}, "
+            f"operation_response={self.operation_response}, "
+            f"kind={self.kind})"[:1024]
+        )
+
+
+class BatchOperationError:
```

Comment:
we would be missing out on the error translations for each of the status codes - which you previously pointed out might be needed since we can't just return http status codes to our users for the responses